libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -226,6 +242,19 @@ void AP_GPS::init(DataFlash_Class *dataflash, const AP_SerialManager& serial_man
     _port[0] = serial_manager.find_serial(AP_SerialManager::SerialProtocol_GPS, 0);
     _port[1] = serial_manager.find_serial(AP_SerialManager::SerialProtocol_GPS, 1);
     _last_instance_swap_ms = 0;
+
+    // Initialise class variables used to do GPS blending
+    memset(&_NE_pos_offset_m, 0, sizeof(_NE_pos_offset_m));
```

Comment:
These are already init'ed to 0 I believe, and failing that should be init'ed in the constructor, which minimizes flash consumption. @tridge might have some more detailed comments on that one, I admit I'm reciting what I've been informed of here.